django/conf/locale/__init__.py

```diff
@@ -386,6 +386,12 @@
         'name': 'Marathi',
         'name_local': 'मराठी',
     },
+    'ms': {
+        'bidi': False,
+        'code': 'ms',
+        'name': 'Malay',
+        'name_local': 'Bahasa Melayu',
```

Comment:
```suggestion
        'name_local': 'Melayu',
```

Reply:
`name_local` should be `Bahasa Melayu`, it's the same case as `Bahasa Indonesia`.

Reply:
Thanks for confirmation.